### ISSUE TITLE
Harvest Ethnologue Maps

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/translationDatabaseWeb#57]
The Ethnologue makes low-res, watermarked language maps available for free download on their website, located at consistent URLs based on the ISO 3166 country code (VN = Vietnam, etc.), e.g. http://www.ethnologue.com/country/VN/maps

Given that we know the full list of ISO 3166 country codes, and if the structure is consistent , we need a script to download all the maps available for every country and show them in tD.

Then, generate a PDF with a TOC for each country code (and name) and map thereof.

**Comment by swilcox** [comment]
Is there a licensing issue that would need to settled regarding part 3:

`and (3) no more than three of these graphics are used in any one publication or internet site`

**Comment by jag3773** [comment]
Yes, you are right.  I don't think we noticed the separate maps license (at http://www.ethnologue.com/map-licenses).  Let's put this on hold for now.
